### PR TITLE
Client side claim and unclaim is added

### PR DESCRIPTION
**Pull request by HarshOB**
Issue id #442

added Client side claim and un-claim of instance.

screenshot from 2018-09-12 17-52-55
screenshot from 2018-09-12 17-53-05

### CODE REVIEW

File: client/src/pages/user/SchemaList.vue [hunk 1]
```diff
@@ -44,7 +44,23 @@
         </div>
       </div>
     </div>
-    <div>
+    <div v-if="role === 'admin'">
```
Comment (chiragob) [inline client/src/pages/user/SchemaList.vue]: Do NOT hardcode the roles

File: client/src/pages/user/SchemaList.vue [hunk 2]
```diff
@@ -255,6 +277,149 @@
...
+                    shape: 'circle',
+                    icon: 'ios-play'
+                  },
+                  domProps: {
+                    title: 'Start'
+                  },
+                  on: {
+                    'click': async () => {
+                      this.$Loading.start()
+                      let indexFind = _.findIndex(this.instanceEntries, (o) => { return o.id === params.row.id })
```
Comment (chiragob) [inline client/src/pages/user/SchemaList.vue]: Why is findIndex needed?

File: client/src/pages/user/SchemaList.vue [hunk 3]
```diff
@@ -255,6 +277,149 @@
...
+                      let currentObj = this.flowzData.processList[this.instanceEntries[indexFind].currentStatus]
```
Comment (chiragob) [inline client/src/pages/user/SchemaList.vue]: Objects that are not changing should be const

File: client/src/pages/user/SchemaList.vue [hunk 4]
```diff
@@ -255,6 +277,149 @@
...
+                      let values = {
+                        id: this.flowzData.schema,
+                        item: this.instanceEntries[indexFind],
+                        formName: currentObj.name,
+                        currentState: currentObj.id,
+                        flowzData: this.flowzData,
+                        formData: params.row.data
+                      }
+                      this.$Loading.finish()
+                      await this.$emit('setValues', values)
```
Comment (chiragob) [inline client/src/pages/user/SchemaList.vue]: Never use await in frontend code

File: client/src/components/SchemaView.vue [hunk 5]
```diff
@@ -1022,6 +1035,8 @@ export default {
   },
   mounted () {
     this.init()
+    this.client = this.$store.state.role === 2 ? true : false
```
Comment (chiragob) [inline client/src/components/SchemaView.vue]: No hardcoding values in code

File: client/src/pages/user/SchemaList.vue (hunk 1 above)
Comment (HarshOB) [reply client/src/pages/user/SchemaList.vue]: Changed in next commit

**Pull request merged**